scripts/check-runtime-versions.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Required minimum versions.
+REQUIRED_GO_VERSION=$(grep 'version:' .promu.yml | awk '{print $2}')
+REQUIRED_NODE_VERSION=$(cat web/ui/.nvmrc | tr -d '\r' | sed 's/v//')
```

Comment:
```suggestion
MIN_NODE_VERSION=$(cat web/ui/.nvmrc | tr -d '\r' | sed 's/v//')
```
?

Reply:
also @juliusv could confirm whether `web/ui/.nvmrc` is the best way to get the min version for node.

Reply:
Sure, replaced REQUIRED with MIN